app/controllers/my-sessions/view.js

```diff
@@ -13,6 +13,16 @@ export default class extends Controller {
 
     return true;
   }
+  
+  @computed('model.event.speakersCall.endsAt')
+  get isCfsOpen(){
+    let cfsEndsAt = this.get('model.event.speakersCall.endsAt');
+    if(cfsEndsAt  < moment()) {
+      return false;
+    }
+    
+    return true;
+    }
```

Comment:
Try
```js
return (this.model.event.speakersCall.endsAt < moment())

Reply:
@maze-runnar

Reply:
@kushthedude i used this -
```
@computed('model.event.speakersCall.endsAt')
  get isCfsOpen(){
    return (this.model.event.speakersCall.endsAt < moment());
    }
```
and http://localhost:4200/my-sessions/s/25 broke. 
![Screenshot from 2020-01-07 00-07-08](https://user-images.githubusercontent.com/56407566/71839661-b3c49080-30e1-11ea-8a4a-eb66565b4ce7.png)

Reply:
it's not working as supposed .